enrollment.js

```diff
@@ -0,0 +1,26 @@
+window.easyEdu = {
+  enroll: (authKey, params, sucessCallback, errorCallback) => {
+    if (!authKey || typeof authKey !== "string") throw "Auth key for EasyEdu class is missing, please pass it as first argument, it must be a string."
+    if (!params || typeof params !== "object") throw "Params object is missing, please pass it as second argument, it must be an object."
```

Comment:
you can create a function `validateParam(name, value, kind)` it should twrow a message for missing param, and another message for invalid type param.